README.md

```diff
@@ -148,21 +134,22 @@ metadata:
   name: gold
 provisioner: block.csi.ibm.com
 parameters:
-  #SpaceEfficiency: <VALUE>    # Optional: Values applicable for Storwize are: thin, compressed, or deduplicated
-  pool: <VALUE_POOL_NAME>
+  SpaceEfficiency: deduplicated
+  pool: gold
 
-  csi.storage.k8s.io/provisioner-secret-name: <VALUE_ARRAY_SECRET>
-  csi.storage.k8s.io/provisioner-secret-namespace: <VALUE_ARRAY_SECRET_NAMESPACE>
-  csi.storage.k8s.io/controller-publish-secret-name: <VALUE_ARRAY_SECRET>
-  csi.storage.k8s.io/controller-publish-secret-namespace: <VALUE_ARRAY_SECRET_NAMESPACE>
+  csi.storage.k8s.io/provisioner-secret-name: svc-array
+  csi.storage.k8s.io/provisioner-secret-namespace: csi-ns
+  csi.storage.k8s.io/controller-publish-secret-name: svc-array
+  csi.storage.k8s.io/controller-publish-secret-namespace: csi-ns
 
-  csi.storage.k8s.io/fstype: xfs   # Optional: Values ext4/xfs. The default is ext4.
+  csi.storage.k8s.io/fstype: xfs   # Optional. values ext4\xfs. The default is ext4.
+  volume_name_prefix: demo         # Optional.
```

Comment:
I thought we want to say something about ds8k here

Reply:
what?

Reply:
something about maximum length / that it's being ignored?

Reply:
prefix length is limited but not ignored.
For each storage the limit is different.  I can't thin of any short comment that can explain that, This explanation is for user guide.

Reply:
ok

Reply:
So approve this one please: https://github.com/IBM/ibm-block-csi-driver/pull/161
The missing changes are here.